postinstall.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+if (
+  process.platform !== 'win32' &&
```

Comment:
I wrote an equivalent setup.bat but there is a test failing on a package that prevents lerna from working (it just hangs), once fixed that I can just add it, this will allow using jest in the meantime